core/src/main/java/io/seata/core/constants/ConfigurationKeys.java

```diff
@@ -425,4 +425,9 @@
      * The constant DISABLE_GLOBAL_TRANSACTION.
      */
     public static final String DISABLE_GLOBAL_TRANSACTION = SERVICE_PREFIX + "disableGlobalTransaction";
+
+    /**
+     * The constant SQL_PARSER_TYPE.
+     */
+    public static final String SQL_PARSER_TYPE = "sqlparser.type";
```

Comment:
suggest replace "sqlparser.type" with CLIENT_RM_PREFIX+"sqlparser.type", and please add it in script module.